src/main/webapp/resources/js/pages/projects/share/index.js

```diff
@@ -14,17 +34,75 @@ import store from "./store";
  * @constructor
  */
 function ShareApp() {
+  const [step, setStep] = useState(0);
```

Comment:
Why are we using just `useState` here instead of `React.useState` like we have been everywhere else?

Reply:
Really 6 of one, half dozen of the other.  I will update it, but again, don't think to critical.

Reply:
Yeah we have been all over the place with them. Let's try to just use React.useState whenever we need it. Thanks :)